app.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import aws_cdk as cdk
+
+from bde_fdw_rds.bde_fdw_rds_stack import BdeFdwRdsStack
```

Comment:
That's three nested names with almost the same name as the repo. Would it be possible to flatten and rename these to avoid naming them the same (or a "subset" of) their container? Something like `from stack import Stack`? Or even inlining the stack in this file?